packages/app/src/embed/components/Header/elements.js

```diff
@@ -82,6 +67,13 @@ export const Title = styled.div`
   overflow: hidden;
   flex: 1;
 
+  /* compatibility mode for the redesign */
```

Comment:
🔖 this bit exists only for release 1, the final redesign wouldn't have this header